.travis.yml

```diff
@@ -1,63 +1,51 @@
 language: cpp
 
+sudo: required
+
 cache: ccache
 
 matrix:
   include:
+    # Ubuntu 14.04 + GCC 4.8 + Qt 5.2 + Doxygen Generation (all packages from official Ubuntu repositories)
     - os: linux
+      dist: trusty
       compiler: gcc
-      env: QT_BASE=52 BUILD_DOXYGEN=true
-    - os: linux
-      compiler: gcc
-      env: QT_BASE=53 BUILD_DOXYGEN=false
-    - os: linux
-      compiler: gcc
-      env: QT_BASE=54 BUILD_DOXYGEN=false
-    - os: linux
-      compiler: gcc
-      env: QT_BASE=55 BUILD_DOXYGEN=false
-    - os: linux
-      compiler: clang
-      env: QT_BASE=52 BUILD_DOXYGEN=false
+      env: BUILD_DOXYGEN=true QT_BASE="trusty"
+    # Ubuntu 14.04 + Clang + Latest Qt from beineri PPA (https://launchpad.net/~beineri)
     - os: linux
+      dist: trusty
       compiler: clang
-      env: QT_BASE=55 BUILD_DOXYGEN=false
+      env: BUILD_DOXYGEN=false QT_BASE="qt57" QT_PPA="ppa:beineri/opt-qt57-trusty"
+    # OS X + GCC
     - os: osx
       compiler: gcc
       env: BUILD_DOXYGEN=false
+    # OS X + Clang
     - os: osx
       compiler: clang
       env: BUILD_DOXYGEN=false
   fast_finish: true
 
 env:
   global:
-    - secure: "QW5QTwmLqovoDDrwVGXIOh+jd+ArwWSAy2P1U5xDW3C7oE5lgHQH3bqStEv63aOxvpN9cgqjroXQfCpKz+nNO7Rur0MY7UXCUxk0VizCOn4CUg7K7XeHTJEvzGRke2j7QUNigE5GTu9ABAeSN/AfHAtWVCFrecMINqTLWWC3pB/dTalCX6NoQb2f1z2nbdHa+suT8NoUo14eXrQfIZ9xnVrQ7NYuMXyb5A0akyQSq1oNb+5yj1o1wdmaNaCkalGyN2Oj/aD8urSU65sahwOQlhCwnxKDVuFqhcgV4ux6k42B7VJuVhxC2j71vCS4Kfka00/WVTNJXmdDxB873o+GqGVElvc5QeimQX34Zw2b+u77teZku4y33wm1wazh6mGh13/4EzA325rqjaxlge4EE5Bzq5qIhmaLspwob1EQD/waEhSDptNCK5bgP+OBtGi1f8vAnjlhh/3U10oWtAV1faV88zx9q0mWb4yYyoLP6D3ViglL62E9hKpwL2uXpGeb6yTCHfcpNMqkwFwYFt+Dj5vJ9RG4kmlsg4oXUn47zCOwj2ynOR24q9xgR+7d2Zj2vzfE/PPWbn8UGO6VcZtn6CpBUn6kMpfh5BcvBA06aFh/neW9WxAe9W7X6w1n1whpkTE2ro3iJDlVDiY+QTlCuZJg6EvLmKuHMf6oKM5WNC0="
+    # variable $DOXYGEN_ACCESS_TOKEN (used to push the doxygen output to https://github.com/LibrePCB/LibrePCB-Doxygen)
+    - secure: "iuBMC/IEaPXY+gQY3BpUrgmLZsD5ldqXnEK01QgqraPRYDmKSN3W9Aw9MR3M3BrxRwnpofy9EFBEV7Z0l5LLelIzUxCZHVfWly6d2BEWSdgrsVOfbu3yPOfd4oen9VOv5sRvbpVEfsLsFxrEvVwSOjX3NZvd8bhhuJSFV7yR3Yr9p2clYrl2e8oI57ocVwA4f452hnpZWuxL137/OFIfZUU/FIwvR+jipvBTB+GyoQpfU0gtue+w4GElw0aVdzZ3HHEPNu4oH6CBIHoKYFP57Ftr7Oh5PtJ6QnCeRb+j7tEeD28naHGjc3KUxdMvbAku9Ltb6B0fypHqFnlqsQFETgJgO1VYlJocLcUHJkOSx2o/Nx5z7juVmGaFJWWMgLNDunO4ZG9oBpTW9B62k7zQ0qHZYT3Ci35/OMkrRxsoTDSj5k6MBMxiImJ7Kv6VrYSl0jnecWK+RM+DA53W4IKrMNnymGPTLK0z6NtZnPOXDGIl6zRpUQqlbZFEBrNT5/domqf04OWlSnP2bjW9XfoEnI7Ww42GTgFMAt5lYAajbdcgZxj6Zm1s/wFGEtHHUcZFgJe+kjA4XCg7pt3Mvtov/GTzjqZdsuiWuGCg1cQIABeugj8M/QYkqWmDCV0o8gNthHyR6IEdhSRDO5zC5nonClojJpWv6NYYTppZjUjBY3E="
```

Comment:
Because of a security issue: https://github.com/LibrePCB/LibrePCB/pull/101/commits/5843f58a41844caa14634b9e979f6f264c8b0a20 ;)